frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/Platform.java

```diff
@@ -66,14 +66,15 @@ public String getLongNodeFilename(String nodeVersion) {
 
     public String getNodeDownloadFilename(String nodeVersion) {
         if(isWindows()) {
+			int versionCompare = nodeVersion.compareTo("v4.0.0");
             if(architecture == Architecture.x64){
-                if (nodeVersion == "v4.0.0") {
+				if (versionCompare >= 0) {
```

Comment:
`"v4.0.0".equals( nodeVersion )` would suffice without diff verbosity.
It seems there are some spacing changes here.